### ISSUE TITLE
unknown field [must_exist] for post _aliases remove action 

### ISSUE THREAD

**Issue by payalm08** [opensearch-project/OpenSearch#2949]
**Describe the bug**
When executing api post _alias with remove action with ""must_exist": true" in request, getting error unknown field [must_exist]

**To Reproduce**

```
POST _aliases 
{
   "actions": [
    {
      "remove": {
        "index": "testIndex",
        "aliases": "testAlias",
        "must_exist": true
      }
    }
  ]
}

```
Error

```
{
  "error" : {
    "root_cause" : [
      {
        "type" : "x_content_parse_exception",
        "reason" : "[7:9] [remove] unknown field [must_exist]"
      }
    ],
    "type" : "x_content_parse_exception",
    "reason" : "[7:23] [aliases] failed to parse field [actions]",
    "caused_by" : {
      "type" : "x_content_parse_exception",
      "reason" : "[7:23] [alias_action] failed to parse field [remove]",
      "caused_by" : {
        "type" : "x_content_parse_exception",
        "reason" : "[7:9] [remove] unknown field [must_exist]"
      }
    }
  },
  "status" : 400
}
```

**Expected behavior**
Expected response

```
{
  "acknowledged" : true
}

```

**Host/Environment (please complete the following information):**
 - Version - OS 1.2

Reported  https://github.com/elastic/elasticsearch/issues/62642

**Comment by dblock** [comment]
Looks like this isn't implemented. Would gladly accept a PR implemented from scratch in OpenSearch.

**Comment by gaobinlong** [comment]
The parameter `must_exist` exists in the [code](https://github.com/opensearch-project/OpenSearch/blob/6c980bc9079c383569613a005937eedb5c97cacc/server/src/main/java/org/opensearch/action/admin/indices/alias/IndicesAliasesRequest.java#L115) but the functionality is not complete yet, I'm trying to fix this issue.

**Comment by gaobinlong** [comment]
I've created a [PR](https://github.com/opensearch-project/OpenSearch/pull/11210) for this issue, could anyone help to review?

**Comment by kiranprakash154** [comment]
Hi, are we on track for this to be released in 2.12 ?

**Comment by gaobinlong** [comment]
> Hi, are we on track for this to be released in 2.12 ?

Yes, the bug fix will be released in 2.12.0.

**Comment by gaobinlong** [comment]
Closed by https://github.com/opensearch-project/OpenSearch/pull/11210.
